init.lua

```diff
@@ -614,8 +614,8 @@ require('lazy').setup({
         -- Some languages (like typescript) have entire language plugins that can be useful:
         --    https://github.com/pmizio/typescript-tools.nvim
         --
-        -- But for many setups, the LSP (`tsserver`) will work just fine
-        -- tsserver = {},
+        -- But for many setups, the LSP (`ts_ls`) will work just fine
+        -- ts_ls = {},
```

Comment:
Shouldn't this remain `tsserver` until Mason updates to `ts_ls`? Otherwise, we'd be looking for `ts_ls` in Mason and then your fix on line 656 doesn't do anything?

Reply:
You're right, I missed that 😵

It should remain `tsserver` in case anyone uncomments it